GeneratorInterface/Pythia8Interface/interface/Py8InterfaceBase.h

```diff
@@ -51,6 +53,13 @@ namespace gen {
 	 unsigned int                   maxEventsToPrint;
          HepMC::IO_AsciiParticles*      ascii_io;
 
+         // EvtGen plugin
+         //
+         bool useEvtGen;
+         EvtGenDecays* evtgenDecays;
+         string evtgenDecFile;
```

Comment:
no idea. but it must be the case for this to compile, no? [I’m not suggesting to fix the header, just to protect against it eventually being fixed]

> On Aug 8, 2015, at 11:16 AM, Josh Bendavid <notifications@github.com> wrote:
> 
> In GeneratorInterface/Pythia8Interface/interface/Py8InterfaceBase.h:
> 
> > @@ -51,6 +53,13 @@ namespace gen {
> >  	 unsigned int                   maxEventsToPrint;
> >           HepMC::IO_AsciiParticles*      ascii_io;
> >  
> > +         // EvtGen plugin
> > +         //
> > +         bool useEvtGen;
> > +         EvtGenDecays* evtgenDecays;
> > +         string evtgenDecFile;
> 
> Sorry, which header has "using std" ?
> 
> —
> Reply to this email directly or view it on GitHub.
>